src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -672,4 +760,70 @@ mod tests {
             assert_eq!(output, expected, "lhs={:?}, rhs={:?}", lhs, rhs);
         }
     }
+
+    #[test]
+    fn test_int_divide_int() {
+        let test_cases = vec![
+            (13, 11, Some(1), false),
+            (13, -11, Some(-1), false),
+            (-13, 11, Some(-1), false),
+            (-13, -11, Some(1), false),
+            (33, 11, Some(3), false),
+            (33, -11, Some(-3), false),
+            (-33, 11, Some(-3), false),
+            (-33, -11, Some(3), false),
+            (11, 0, None, false),
+            (-11, 0, None, false),
+            (-3, 5, Some(0), false),
+            (3, -5, Some(0), false),
+            (std::i64::MIN + 1, -1, Some(std::i64::MAX), false),
+            (std::i64::MIN, 1, Some(std::i64::MIN), false),
+            (std::i64::MAX, 1, Some(std::i64::MAX), false),
+            (std::u64::MAX as i64, 1, Some(std::u64::MAX as i64), false),
+        ];
+
+        for (lhs, rhs, expected, is_err) in test_cases {
+            let output = RpnFnScalarEvaluator::new()
+                .push_param(lhs)
+                .push_param(rhs)
+                .evaluate(ScalarFuncSig::IntDivideInt);
+
+            if is_err {
+                assert!(output.is_err(), "lhs={:?}, rhs={:?}", lhs, rhs);
+            } else {
+                let output = output.unwrap();
+                assert_eq!(output, expected, "lhs={:?}, rhs={:?}", lhs, rhs);
+            }
+        }
+    }
+
+    #[test]
+    fn test_int_divide_decimal() {
+        let test_cases = vec![
+            ("11.01", "1.1", "10", false),
+            ("-11.01", "1.1", "-10", false),
+            ("11.01", "-1.1", "-10", false),
+            ("-11.01", "-1.1", "10", false),
+            ("123", "", "", false),
```

Comment:
Please write None directly.

Reply:
> Please write None directly.

The vector is like `Tuple(lhs, rhs, expected, is_err)`, should I write everything(lhs, rhs, expected, is_err) directly?

Reply:
You can refer to other test cases. Simply `None` or `Some(...) ` is clear.